pyiron_gui/project/project_browser.py

```diff
@@ -154,14 +363,21 @@ def _busy_check(self, busy=True):
 
     def _update_files(self):
         # HDF and S3 project do not have list_files
+        node_filter = ['NAME', 'TYPE', 'VERSION', 'HDF_VERSION']
```

Comment:
For this PR having `node_filter` defined here is totally fine, but it does flag for me that we need this defined somewhere more accessible, e.g. `pyiron_base.constants` or something like that. I'm just imagining that we update `'TYPE'` to `'STYLE'` or something silly like that and then it breaks random parts of the code that were depending on a particular hard-coded naming convention. That's an issue for base though, not gui.